RPIPs/RPIP-46.md

```diff
@@ -29,91 +29,121 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
+   6. `node_operator_commission_share`, `node_operator_commission_share_council_adder`, and `new_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
+2. `voter_share` SHALL no longer be a pDAO setting. the pDAO will not be able to vote changes to it and changes will rely on the method described below.
+3. Updating `voter_share`:
    1. A permissionless function SHALL be available to update `voter_share`
    2. It MUST revert if it's been called within the last 45 days
-   3. It MUST revert if the total RPL eligible to vote is between `vote_eligible_target_min` and `vote_eligible_target_max` (inclusive)
+   3. It MUST revert if the total RPL eligible to vote relative to the circulating supply of RPL is between `vote_eligible_target_min` and `vote_eligible_target_max` (inclusive)
+      1. RPL eligible to vote includes both megapool staked RPL and legacy staked RPL
+      2. The circulating supply of RPL is the total supply of RPL minus RPL owned by the protocol (eg, in treasury or an LP as a result of RPIP-50) 
    4. If <`vote_eligible_target_min` of total RPL is eligible to vote and the function succeeds:
       1. `voter_share` is increased to `voter_share * (1+voter_share_relative_step)`
-      2. `surplus_share` is decreased by the difference between the old and new `voter_share`
-         1. If this would reduce `surplus_share` below 0%, the function call MUST revert 
+      2. `new_share` is decreased by the difference between the old and new `voter_share`
+         1. If this would reduce `new_share` below 0%, the function call MUST revert 
    5. If >`vote_eligible_target_max` of total RPL is eligible to vote and the function succeeds:
       1. `voter_share` is decreased to `voter_share / (1+voter_share_relative_step)`
-      2. `surplus_share` is increased by the difference between the old and new `voter_share`
+      2. `new_share` is increased by the difference between the old and new `voter_share`
    6. Because this involves _voters_ modifying `voter_share`, there is an acknowledged conflict of interest here. As a result, changing this method of "Updating `voter_share`" SHALL require a supermajority vote with at least 75% of the vote in support of any change.
-2. The pDAO SHALL be able to update `voter_share_relative_step`, `vote_eligible_target_min`, and `vote_eligible_target_max` by vote; however, it SHALL require a supermajority vote with at least 75% of the vote in support of any change.
-3. The initial settings SHALL be:
+4. The pDAO SHALL be able to update `voter_share_relative_step`, `vote_eligible_target_min`, and `vote_eligible_target_max` by vote; however, it SHALL require a supermajority vote with at least 75% of the vote in support of any change.
+5. The initial settings SHALL be:
    1. `voter_share_relative_step`: 15%
    2. `vote_eligible_target_min`: 55%
    3. `vote_eligible_target_max`: 65%
 
 ## Optional heuristics
```

Comment:
Given how long this is getting, I would like to advocate again for moving these sections to a supporting document and linking here. I guess alternatively you could use the click-to-expand functionality used here: <https://rpips.rocketpool.net/RPIPs/RPIP-54#outstanding-discrepancies>

Reply:
If / once this become active, this could be undone.

Reply:
I like the click-to-expand...

Reply:
Click to expand works for me